cstrml/market/src/lib.rs

```diff
@@ -1166,7 +1166,8 @@ impl<T: Config> Module<T> {
                 } else {
                     let total_amount = file_info.amount + file_info.prepaid;
                     if !Self::maybe_reward_merchant(who, &total_amount, true) {
-                        T::Currency::transfer(&Self::storage_pot(), &Self::reserved_pot(), total_amount, KeepAlive).expect("Something wrong during transferring");
+                        // This should not have error => discard the result
+                        let _ = T::Currency::transfer(&Self::storage_pot(), &Self::reserved_pot(), total_amount, KeepAlive);
```

Comment:
This is one special case since it's not possible to throw out error and I don't want to make a huge refractor